docs/en/Running-HBase-on-Alluxio.md

```diff
@@ -0,0 +1,131 @@
+---
+layout: global
+title: Running Apache HBase on Alluxio
+nickname: Apache HBase
+group: Frameworks
+priority: 2
+---
+
+This guide describes how to run [Apache HBase](http://hbase.apache.org/), so
+that you can easily store HBase tables into Alluxio at various storage level.
+
+# Prerequisites
+
+The prerequisite for this part is that you have
+[Java](Java-Setup.html). Alluxio cluster should also be
+set up in accordance to these guides for either [Local Mode](Running-Alluxio-Locally.html) or
+[Cluster Mode](Running-Alluxio-on-a-Cluster.html).
+
+Please follow the guides for setting up HBase on
+[Apache HBase Configuration](https://hbase.apache.org/book.html#configuration).
+
+# Configuration
+
+Apache HBase allows you to use Alluxio through a generic file system wrapper for the Hadoop file system.
+Therefore, the configuration of Alluxio is done mostly in HBase configuration files.
+
+#### Set property in `hbase-site.xml`
+
+You need to add the following three properties to `hbase-site.xml` in your HBase installation `conf` directory:
+
+Tips:You do not need to create the /hbase directory in Alluxio, HBase will do this for you.
+
+```xml
+<property>
+  <name>fs.alluxio.impl</name>
+  <value>alluxio.hadoop.FileSystem</value>
+</property>
+<property>
+  <name>fs.alluxio-ft.impl</name>
+  <value>alluxio.hadoop.FaultTolerantFileSystem</value>
```

Comment:
ok, I have fix the problem, just wait for a few minutes before a standby master becomes active!!